default.nix

```diff
@@ -0,0 +1,7 @@
+{ pkgs ? import <nixpkgs> {} }:
+let
+  result = import /home/dmj/miso {};
```

Comment:
Is there a way of setting this to the $HOME env variable instead of having a static location?

PS. I've never used nix before hence my cluelessness.